.github/workflows/push.yml

```diff
@@ -0,0 +1,62 @@
+name: Unit tests & Build Testapp
+
+on: ['push', 'pull_request']
+
+jobs:
+
+  flake8:
+    name: Flake8 tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout python-for-android
+      uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1.1.0
+      with:
+        python-version: 3.7
+    - name: Run flake8
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox>=2.0
+        tox -e pep8
+
+  linting:
+    name: Pytest [Python ${{ matrix.python-version }} | ${{ matrix.os }}]
+    needs: flake8
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+        os: [ubuntu-latest, macOs-latest]
+    steps:
+    - name: Checkout python-for-android
+      uses: actions/checkout@master
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1.1.0
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Tox Python 3
+      # ignores test_pythonpackage.py since it runs for too long
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox>=2.0 pyOpenSSL coveralls
+        tox -e py3 -- tests/ --ignore tests/test_pythonpackage.py
+    - name: Send Coverage to coveralls
+      run: |
+        coveralls
```

Comment:
Yeah, I like it, this way the dilemma of deciding when we should send the report to coveralls it will be solved, since `CI` environment variable will only exist for travis, so no report will be send with gh-actions unless we change that in the `makefile` by the gh-actions equivalent (which is `GITHUB_ACTIONS`...for the record :wink:).

Thanks again @AndreMiras :smile:

Reply:
Nice it looks simpler. Good to know for the env variable